core/chains/evm/txmgr/orm.go

```diff
@@ -232,6 +233,22 @@ func (o *orm) FindEthTxWithAttempts(etxID int64) (etx EthTx, err error) {
 	return etx, errors.Wrap(err, "FindEthTxWithAttempts failed")
 }
 
+func (o *orm) FindEthTxAttemptConfirmedByEthTxIDs(ids []int64) ([]EthTxAttempt, error) {
+	var attempts []EthTxAttempt
+	err := o.q.Transaction(func(tx pg.Queryer) error {
+		if err := tx.Select(&attempts, `SELECT eta.*
+		FROM eth_tx_attempts eta
+			join eth_receipts er on eta.hash = er.tx_hash where eta.eth_tx_id = ANY($1) ORDER BY eta.gas_price DESC, eta.gas_tip_cap DESC`, ids); err != nil {
```

Comment:
Why select multiple attempts? We only care about the "first" one and by design there will only ever be zero or one anyway.

Reply:
single attempts for multiple transaction ids - so 1 for every transaction id in `ids`